### PR TITLE
Code Quality: Use `LibraryImportAttribute` at undocumented EntryPoint

### PR DESCRIPTION
**Pull request by gumbarros**
**Resolved / Related Issues**
- [X] Were these changes approved in an issue or discussion with the project maintainers? In order to prevent extra work, feature requests and changes to the codebase must be approved before the pull request will be reviewed. This prevents extra work for the contributors and maintainers.
   Part of #15000 

**Validation**
How did you test these changes?
- [X] Did you build the app and test your changes?
- [ ] Did you check for accessibility? You can use Accessibility Insights for this.
- [ ] Did you remove any strings from the en-us resource file?
   - [ ] Did you search the solution to see if the string is still being used? 
- [ ] Did you implement any design changes to an existing feature?
   - [ ] Was this change approved?
- [ ] Are there any other steps that were used to validate these changes?
   1. Open app ...
   2. Click settings button ...

**Additional Context**
[#865 is undocumented](https://github.com/microsoft/CsWin32/issues/1170), so it's impossible to use CsWin32.

### CODE REVIEW

**Review by 0x5bfa — APPROVED** [review]

File: src/Files.App/Helpers/Environment/ElevationHelpers.cs
Comment (yaira2) [inline src/Files.App/Helpers/Environment/ElevationHelpers.cs:1]: Are we actively using this helper? I think it can be removed.

Comment (0x5bfa) [reply src/Files.App/Helpers/Environment/ElevationHelpers.cs:1]: On board

Comment (gumbarros) [reply src/Files.App/Helpers/Environment/ElevationHelpers.cs:1]: Agree, but just tomorrow at night bros

Comment (gumbarros) [reply src/Files.App/Helpers/Environment/ElevationHelpers.cs:1]: @yaira2 good night!

The `class is used in these places`

image

The Win32 method in just these places:
image

Can I remove the class and update the callers with the `if` or leave as is?

Comment (yaira2) [reply src/Files.App/Helpers/Environment/ElevationHelpers.cs:1]: I mixed it up with a different method, we can keep this as in then.

**Pull request merged**